packages/manager/modules/dbaas-logs/src/logs/detail/encryption-keys/home/logs-encryption-keys-home.html

```diff
@@ -0,0 +1,81 @@
+<section class="log-encryption-keys">
+    <oui-back-button data-on-click="$ctrl.goToHomePage()">
+        <span data-translate="logs_encryption_keys_title"></span>
+    </oui-back-button>
+
+    <p data-translate="logs_encryption_keys_description"></p>
+
+    <oui-datagrid
+        id="encryption-keys-datagrid"
+        data-rows-loader="$ctrl.loadEncryptionKeys($config)"
+    >
+        <oui-datagrid-topbar>
+            <button
+                type="button"
+                class="oui-button oui-button_secondary"
+                data-ng-click="$ctrl.goToAddPage()"
+            >
+                <span
+                    class="oui-icon oui-icon-add pr-1"
+                    aria-hidden="true"
+                ></span>
+                <span data-translate="logs_encryption_keys_add"></span>
+            </button>
+            <oui-spinner
+                size="s"
+                data-ng-if="$ctrl.deleteEncryptionKeyLoading"
+            ></oui-spinner>
+        </oui-datagrid-topbar>
+        <oui-datagrid-column
+            data-title="::'logs_encryption_keys_col_title' | translate"
+            data-property="title"
+            data-type="string"
+            data-sortable
+            data-searchable
+            data-filterable
+        ></oui-datagrid-column>
+        <oui-datagrid-column
+            data-title="::'logs_encryption_keys_col_fingerprint' | translate"
+            data-property="fingerprint"
+            data-type="string"
+            data-sortable
+            data-searchable
+            data-filterable
+        >
+        </oui-datagrid-column>
+        <oui-datagrid-column
+            data-title="::'logs_encryption_keys_col_uid' | translate"
+            data-property="uid"
+            data-type="string"
+            data-sortable
+            data-searchable
+            data-filterable
+        >
+            <span class="ellipsis" data-ng-bind="$row.uid"></span>
```

Comment:
**nitpick:** Utility classes from the bootstrap can be used.
For more info, you can refer https://getbootstrap.com/docs/4.0/utilities/text/#text-wrapping-and-overflow

```suggestion
            <span class="d-inline-block text-truncate" data-ng-bind="$row.uid"></span>
```

Reply:
Fixed, I didn't knew about this utility class, thanks.
However it works well on the `span`, the the `oui-datagrid-column` keep the original size. And no class seems to be appliable on it

Reply:
You have to specify the width of the `span`. Then it works I guess.